app/src/main/java/com/github/onlynotesswent/ui/overview/EditNote.kt

```diff
@@ -367,78 +357,20 @@ fun PdfCard(
       })
 }
 
-/** Displays a screen indicating that the user was not found. */
+/**
+ * Displays a screen indicating that the user was not found.
+ *
+ * @param errorText The error message to display.
+ */
 @Composable
-fun UserNotFoundEditScreen() {
+fun ErrorScreen(errorText: String) {
```

Comment:
Nice idea making a common error screen, we could probably make this common composable for the whole app and also add a TAG parameter

Reply:
Totally agree, I'll leave this for a future PR where we can create utility classes for UI, tests, MVVM, which will make coding them more uniform and reduce redundant code

Reply:
Ok perfect